x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.scss

```
@@ -0,0 +1,9 @@
+.relevanceTuningForm {
+  .relevanceTuningForm__item {
+    width: 100%;
+  }
+
+  .relevanceTuningForm__panel + .relevanceTuningForm__panel {
```

Same here, I think we can remove the selector nesting

```suggestion
  &__panel + &__panel {
```

---

https://github.com/elastic/kibana/pull/90621/commits/805f5afc952c6ab0dc297308aa9b813c8629aeaf